main.py

```
@@ -0,0 +1,134 @@
+@app.delete("/posts/{post_id}/likes")
+async def delete_like(post_id: int):
+    pass
+
+# check like status
+@app.get("/posts/{post_id}/likes}")
+async def get_likes_status(post_id: int):
+    pass
+
+# post list I liked
+@app.get("/posts")
+async def get_posts_liked(liked: bool, page: int = 1, limit: int = 20):
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)